packages/sdk-react/src/components/receivables/InvoiceDetails/CreateReceivable/CreateReceivables.tsx

```diff
@@ -108,6 +108,9 @@ const CreateReceivablesBase = (props: InvoiceDetailsCreateProps) => {
     return <LoadingPage />;
   }
 
+  // eslint-disable-next-line lingui/no-unlocalized-strings
+  const formName = 'Monite-Form-receivablesDetailsForm';
```

Comment:
Could you please refactor it with the `Monite-Form-receivablesDetailsForm-useId()`?

Reply:
You're quite right, the probability of having 2 `Monite-Form-receivablesDetailsForm` identifiers on the page is low. But why think about probabilities when it's better to use [the standard approach](https://react.dev/reference/react/useId#why-is-useid-better-than-an-incrementing-counter) that React offers